src/App.tsx

```diff
@@ -1,15 +1,45 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import './App.scss';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [isFocusedQuery, setIsFocusedQuery] = useState(false);
+  const [highlatedPerson, setHighlatedPerson] = useState<Person>(
+    peopleFromServer[0]
```

Comment:
because of this state you always have selected person and it's not allow you to show "No selected person" initially